conf/map/battle/items.conf

```diff
@@ -119,3 +119,7 @@ bow_unequip_arrow: true
 // How much should rental mounts increase a player's movement speed? (Note 2)
 // Official: 25 (Default)
 boarding_halter_speed: 25
+
+// Allow to use items when the storage is open?
+// Official: false (Default)
+storage_use_item: false
```

Comment:
Change to storage_use_cash_item ?

Reply:
Hmm i was misunderstood before, but when i try to use even apple in kRO even my storage is open, it didnt consume the apple.